docs/user/howto/manage-charms.md

```diff
@@ -1,15 +1,17 @@
 (manage-charms)=
-# How to manage charms or bundles
+# How to manage charms
```

Comment:
the title mentions charms only, while the document contents review to both charms and bundles. it's a bit odd to my eyes.

Reply:
My update of the title was a bit aspirational. Basically I want to pull the bundle content out -- it clutters the charm story, plus we're moving away from bundles anyway. I'll do that in another PR.